packages/block-editor/src/components/list-view/style.scss

```diff
@@ -475,22 +475,16 @@ $block-navigation-max-indent: 8;
 		($block-navigation-max-indent - 1);
 }
 
-.block-editor-list-view-leaf:not([aria-level="1"]) {
-	.block-editor-list-view__expander {
-		margin-right: 4px;
-	}
-}
-
 // When updating the margin for each indentation level, the corresponding
 // indentation in `use-list-view-drop-zone.js` must be updated as well
```

Comment:
We'll also need to update the JS equivalent for the nesting level I believe: https://github.com/WordPress/gutenberg/blob/2997bad8c9119307d0a2767581bc39d7466d439b/packages/block-editor/src/components/list-view/use-list-view-drop-zone.js#L65